src/Symfony/Component/HttpKernel/Tests/Debug/TraceableEventDispatcherTest.php

```diff
@@ -39,6 +44,11 @@ public function testStopwatchSections()
         ), array_keys($events));
     }
 
+    /**
+     * Tagged as legacy because of the usage of the deprecated ControllerResolver#getArguments().
+     *
+     * @group legacy
```

Comment:
This does not make sense. the TraceableEventDispatcher is **not** deprecated, so it should not trigger deprecation warnings.

Tests marked as legacy should only be tests covering BC layers, and this is not the case here